crate/server/src/routes/google_cse/operations.rs

```diff
@@ -312,12 +313,14 @@ pub async fn private_key_sign(
     let database_params = kms.get_sqlite_enc_secrets(&req_http)?;
 
     debug!("private_key_sign: validate_tokens");
-    let user = validate_tokens(
+    let roles: &[&str; 1] = &["signer"];
```

Comment:
create a const variable if it is really hardcoded

Reply:
I put an enum in the end - is this OK for you ?